src/ResourceManager/AzureBatch/ChangeLog.md

```diff
@@ -18,6 +18,63 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Added new parameters to `New-AzureRmBatchAccount`.
```

Comment:
Please ensure that any breakign changes are correctly docuemnted int eh 'Upcoming breaking changes' document (this should have happened on deprecation, but please ensure this is the case now.

Reply:
Yeah, it's updated (I had it in the wrong file but moved it now)